src/HttpClientPoolItem.php

```diff
@@ -53,12 +50,12 @@ public function sendRequest(RequestInterface $request)
         }
 
         try {
-            ++$this->sendingRequestCount;
+            $this->incrementRequestCount();
             $response = $this->client->sendRequest($request);
-            --$this->sendingRequestCount;
+            $this->decrementRequestCount();
         } catch (Exception $e) {
             $this->disable();
-            --$this->sendingRequestCount;
+            $this->decrementRequestCount();
```

Comment:
if we where using php 5.5 only, we could use finally for this. but we are on 5.4 it seems :-(